backend/sftp/sftp.go

```diff
@@ -390,6 +394,7 @@ func NewFs(name, root string, m configmap.Mapper) (fs.Fs, error) {
 	}
 
 	keyFile := env.ShellExpand(opt.KeyFile)
+	keyPem := env.ShellExpand(opt.KeyPem)
```

Comment:
Oh sorry I noticed this before but forgot to write a note

Do we need to `ShellExpand` the KeyPem? I think it is unlikely to have any environment variables we need expanding in so probably replace `keyPem` with `opt.KeyPem` below?

Reply:
ah yeah.  :)  ty